helpers/mathjax.js

```diff
@@ -94,21 +99,17 @@ export function loadMathJax(mathJaxConfig) {
 
 	if (mathJaxLoaded) return mathJaxLoaded;
 
-	const loadOptions = ['ui/menu'];
-	if (mathJaxConfig && mathJaxConfig.renderLatex) {
-		loadOptions.push('[tex]/all-packages');
```

Comment:
One benefit of forcing typesetting to queue is that we no longer need to bulk import all of the TeX packages. MathJax will dynamically import whatever packages are necessary based on the TeX its processing and now that we're only processing one html-block at a time, it's guaranteed that we'll have the packages loaded properly before we typeset.